wellcomecollection/wellcomecollection.org
# Stories page articles from Content API

## What does this change?

Gets the stories page articles from the Content API

## How to test

Visit `/stories` and check the five stories at top of the page and the six stories under the 'You may have missed' header at the bottom of the page all render.

(This change reinstates the comics alongside the other stories in the result lists, so there is an expected difference between the results here and what's in prod).

## How can we measure success?

We're less reliant on Prismic

## Have we considered potential risks?

If we _had_ needed to replicate what is currently in prod, it would have been tricky with the Content API currently, because while you _can_ filter (e.g. all articles that _are_ comics), you can't do a negated filter (e.g. all articles that _aren't_ comics). Will make a note of this as something to bear in mind
